docs/functional/cart.md

```diff
@@ -0,0 +1,39 @@
+# Cart
+This article describes functional behavior of a cart. Article shortly summarizes its promo codes, behavior of cart content when user logs in and others.
```

Comment:
```suggestion
This article describes functional behavior of a cart. Article shortly summarizes its promo codes, behavior of cart content when customer logs in and others.
```

Reply:
After agreement, I changed all customers to users.